src/store/modules/todo.js

```diff
@@ -0,0 +1,40 @@
+import axios from "axios";
+
+export default {
+    actions: {
+        async getAll(ctx) {
+            axios.get("http://127.0.0.1:3000")
+                .then(response => {
+                    ctx.commit('setTodo', response.data)
+                });
+        },
+        async addTodo(ctx, title) {
+            axios.post("http://127.0.0.1:3000", {title})
+                .then(response => {
+                    ctx.dispatch('getAll');
+                });
+        },
+        async updateTodo(ctx, data) {
+            console.log(data)
+            axios.patch(`http://127.0.0.1:3000/${data.id}`, {title : data.title})
```

Comment:
I don't like the idea of store being coupled with rest api. Despite vue guide suggests this pattern, how do you unit test actions if needed?